geoh5py/ui_json/enforcers.py

```diff
@@ -348,6 +341,14 @@ class RequiredWorkspaceObjectEnforcer(RequiredEnforcer):
     enforcer_type = "required_workspace_objects"
     validation_error = RequiredWorkspaceObjectValidationError
 
+    def rule(self, value: Any) -> bool:
+        """True if all objects are in the workspace."""
+        validations = [value[k].uid for k in self.validations]
+        return all(k in self.collection(value) for k in validations)

```

Comment:
Same comment as above, feels like we can do the reserve logic

```suggestion
        entities_list = self.validations[geoh5].list_entities_name
        return all(k in entities_list for k in value)

```

Reply:
self.validations doesn't contain a 'geoh5', in fact it's not even a dictionary, it's a set.